### PR TITLE
Enable Goodman-Kruskal lambda

### PR DESCRIPTION
**Pull request by Kucharssim**
Fixes https://github.com/jasp-stats/jasp-issues/issues/1075

Also comes with a verified unit test.

### CODE REVIEW

**Review by vandenman — CHANGES_REQUESTED** [review]
So this looks fine to me. The only thing I noticed is that all Lambdas are always 0 if there is a 2 by 2 design. That could be correct but I don't know this topic well enough.

File: R/contingencytables.R [hunk 1]
```diff
@@ -548,22 +549,48 @@ ContingencyTables <- function(jaspResults, dataset, options, ...) {
...
+  if(ready) {
+    n <- sum(counts.matrix)
+    csum <- colSums(counts.matrix)
+    rsum <- rowSums(counts.matrix)
+    rmax <- apply(counts.matrix, 1, max)
+    cmax <- apply(counts.matrix, 2, max)
+    max.rsum <- max(rsum)
+    max.csum <- max(csum)
+    nr <- nrow(counts.matrix)
+    nc <- ncol(counts.matrix)
```
[the remark this answers is not in the document]
Comment (Kucharssim) [reply R/contingencytables.R:582]: Yeah, my bad

**Comment by Kucharssim** [comment]
> The only thing I noticed is that all Lambdas are always 0 if there is a 2 by 2 design. That could be correct but I don't know this topic well enough.

It's not caused by 2x2 design, but is a general limitation of the coefficient. From Kvålseth (2017):
Screenshot 2020-12-07 at 17 43 37

The example on Wiki also shows this: https://en.wikipedia.org/wiki/Goodman_and_Kruskal%27s_lambda#Weaknesses

**Review by vandenman — APPROVED** [review]

**Comment by TimKDJ** [comment]
Thanks for reviewing @vandenman, it's a little tricky to stay on top of things in my limited time

**Pull request merged**